src/clib/pio_darray.c

```diff
@@ -949,6 +949,10 @@ PIOc_read_darray(int ncid, int varid, int ioid, PIO_Offset arraylen,
         if ((ierr = pio_read_darray_nc(file, iodesc, varid, iobuf)))
             return pio_err(ios, file, ierr, __FILE__, __LINE__);
         break;
+    case PIO_IOTYPE_GDAL:
```

Comment:
Will need #ifdef

Reply:
Hey. Yeah. I tried to set up an #ifdef. It's on the list of to-do